assets/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
Looking at gfx and Servo as an example, the subcrates do not each contain licenses, presumably because the license files found at the repository root apply to all source code in the tree. But then again, IANAL.